cli/cobra.go

```diff
@@ -23,8 +22,7 @@ import (
 
 // setupCommonRootCommand contains the setup common to
 // SetupRootCommand and SetupPluginRootCommand.
-func setupCommonRootCommand(ctx context.Context, rootCmd *cobra.Command) (*cliflags.ClientOptions, *cobra.Command) {
-	rootCmd.SetContext(ctx)
```

Comment:
removed this here since we can use the cobra `ExecuteContext` instead. This prevents us from passing in `ctx` to the plugins through `RunPlugin`.